js/apps/admin-ui/src/realm-settings/user-profile/AttributesGroupForm.tsx

```diff
@@ -110,82 +110,59 @@ export default function AttributesGroupForm() {
-        <FormAccess isHorizontal role="manage-realm">
-          <FormGroup
-            label={t("nameField")}
-            fieldId="kc-name"
-            isRequired
-            helperTextInvalid={t("required")}
-            validated={form.formState.errors.name ? "error" : "default"}
-            labelIcon={
-              <HelpItem helpText={t("nameHintHelp")} fieldLabelId="nameField" />
-            }
+      <PageSection variant="light">
+        <FormProvider {...form}>
+          <FormAccess
+            isHorizontal
+            role="manage-realm"
+            onSubmit={handleSubmit(onSubmit)}
           >
-            <KeycloakTextInput
-              id="kc-name"
-              isReadOnly={!!matchingGroup}
-              {...form.register("name", { required: true })}
+            <TextControl
+              name="name"
+              label={t("nameField")}
+              labelIcon={t("nameHintHelp")}
+              rules={{ required: t("required") }}
+              readOnly={!!matchingGroup}
             />
-            {!!matchingGroup && (
-              <input type="hidden" {...form.register("name")} />
-            )}
-          </FormGroup>
-          <FormGroup
-            label={t("displayHeaderField")}
-            fieldId="kc-display-header"
-            labelIcon={
-              <HelpItem
-                helpText={t("displayHeaderHintHelp")}
-                fieldLabelId="displayHeaderField"
-              />
-            }
-          >
-            <KeycloakTextInput
-              id="kc-display-header"
-              {...form.register("displayHeader")}
+            <TextControl
+              name="displayHeader"
+              label={t("displayHeaderField")}
+              labelIcon={t("displayHeaderHintHelp")}
             />
-          </FormGroup>
-          <FormGroup
-            label={t("displayDescriptionField")}
-            fieldId="kc-display-description"
-            labelIcon={
-              <HelpItem
-                helpText={t("displayDescriptionHintHelp")}
-                fieldLabelId="displayDescriptionField"
-              />
-            }
-          >
-            <KeycloakTextInput
-              id="kc-display-description"
-              {...form.register("displayDescription")}
+            <TextControl
+              name="displayDescription"
+              label={t("displayDescriptionField")}
+              labelIcon={t("displayDescriptionHintHelp")}
             />
-          </FormGroup>
-          <TextContent>
-            <Text component="h2">{t("annotationsText")}</Text>
-          </TextContent>
-          <FormGroup label={t("annotationsText")} fieldId="kc-annotations">
-            <FormProvider {...form}>
+            <TextContent>
+              <Text component="h2">{t("annotationsText")}</Text>
+            </TextContent>
+            <FormGroup label={t("annotationsText")} fieldId="kc-annotations">
+              {" "}
```

Comment:
Looks like some random white-space was inserted here, let's get rid of it.
```suggestion
```

Reply:
done!